docs/teams/proteus.md

```diff
@@ -42,7 +42,7 @@ Our definition of done must be followed for all work delivered. Sections may onl
 ### Software Dev
 - Code review complete
 - Related pull requests merged
-- Continuous integration pipeline written and passing
+- Continuous integration pipeline written, passing and configured to run appropriate test, test_e2e and build jobs
```

Comment:
Can we make this "unit tests, component tests and build jobs"? That will make it more universal to different projects.

Reply:
Don't disagree but should we then rename/reconfigure them in the pipeline as tests currently equates to unit and component with e2e being UI only?

Reply:
e2e == component